packages/target-tauri/runtime-tauri/runtime.ts

```diff
@@ -348,8 +348,8 @@ class TauriRuntime implements Runtime {
   writeClipboardText(text: string): Promise<void> {
     return writeText(text)
   }
-  writeClipboardImage(_path: string): Promise<void> {
-    throw new Error('Method not implemented.20')
+  writeClipboardImage(path: string): Promise<void> {
```

Comment:
@Simon-Laux What values can `path` have? From my experiments, I saw absolute paths but I guess this can also bea  relative blobdir path?

Reply:
blobdir paths are always absolute - at least currently.